README.md

```diff
@@ -210,9 +218,36 @@ interface CacheProfile
      * @return mixed
      */
     public function cacheNameSuffix(Request $request);
+    
+    /*
+     * Return an array with a key and callback that returns a string. Responsecache will
+     * then use these to replace the response content dynamically, we already do this
+     * for the csrf_token in the BaseCacheProfile. Extend this to your liking.
+     */
+    public function replacers(Request $request): array;
 }
 ```
 
+### Injecting dynamic content with Replacers
+If you need inject dynamic content into the cached response, you can register replacers in the config file.
+We add a `CsrfTokenReplacer` by default in the config file.
+
+Replacers must implement the `Replacer` interface.
+
+```php
+/*
+ * Here you may define replacers that dynamically replace content from the response.
+ * Each replacer must implement the ReplacerInterface.
+ */
+'replacers' => [
+    \Spatie\ResponseCache\Replacers\CsrfTokenReplacer::class,
+],
+```
+
+The callback will be called when the response is cached for the first time, 
```

Comment:
Which callback are you referring to here?

Reply:
Was from the old implementation